cavy.js

```diff
@@ -55,6 +56,10 @@ program
     'Swap the index files and start the report server without first building the app'
   )
   .option('-d, --dev', 'Keep report server alive until manually killed')
+  .option(
+    '-bt, --boot-timeout <seconds>',
+    'Set how long the CLI should wait for the RN app to boot (defaults to 20 seconds)'
```

Comment:
To for publishing: we need to add a note about which Cavy version to use here (at the moment not sure what this will be)